module/platform/OpenCR/HardwareIO/src/hardwareio/send_request.cpp

```diff
@@ -0,0 +1,107 @@
+#include <fmt/format.h>
+
+#include "Convert.hpp"
+#include "HardwareIO.hpp"
+
+namespace module::platform::OpenCR {
+
+    using message::platform::RawSensors;
+
+    void HardwareIO::send_servo_request() {
+
+        // Write out servo data
+        // SYNC_WRITE (write the same memory addresses on all devices)
```

Comment:
Nah I was working on only adding relevant devices to the Sync Write packet to make the packets shorter, which I think may be a better solution (although, comes with it's own problems for implementation).

Using BulkWrite may make instruction packets even longer, as the write addresses would need to be specified for every servo (rather than once at the start). Additionally, we wouldn't be able to construct packets easily using the `DynamixelServoWriteData` structs, as write data would be different, which would be a pain to work with.

As it stands, we don't have reason to be incredibly concerned about efficiency (before RoboCup anyway). The loop ran at >100Hz when I last tested it.